src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -158,7 +171,11 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
 
   useEffect(() => {
     async function loadProjects() {
-      if (page !== 'project-list' || !currencyCode) {
+      if (
+        page !== 'project-list' ||
+        !currencyCode ||
+        (projects && projects?.length > 0)
+      ) {
```

Comment:
It is a detailed skeleton not the list skeleton appears after clicking the back button, possibly because rendering the list of projects is taking some time.